### PR TITLE
Updated documentation about how to generate swap instcutions

### PR DESCRIPTION
**Pull request by unordered-set**

### CODE REVIEW

File: docs/2-apis/1-swap-api.md [hunk 1]
```diff
@@ -314,17 +320,19 @@ const {
...
 
-const swapInstruction = new TransactionInstruction({
-  programId: new PublicKey(swapInstructionPayload.programId),
-  keys: swapInstructionPayload.accounts.map((key) => ({
-    pubkey: new PublicKey(key.pubkey),
+const deserializeInstruction = (instruction) => {
+  return new solanaWeb3.TransactionInstruction({
+    programId: new solanaWeb3.PublicKey(instruction.programId),
+    keys: instruction.accounts.map((key) => ({
+      pubkey: new solanaWeb3.PublicKey(key.pubkey),
```
Comment (siong1987) [inline docs/2-apis/1-swap-api.md:327]: can we just use `PublicKey` here?

Comment (siong1987) [reply docs/2-apis/1-swap-api.md:327]: same for everywhere else.

**Review by siong1987 — CHANGES_REQUESTED** [review]
thanks for the commit!

File: docs/2-apis/1-swap-api.md (hunk 1 above)
Comment (unordered-set) [reply docs/2-apis/1-swap-api.md:327]: oh, sorry, just pasted from my project directly, ofc

**Comment by unordered-set** [comment]
Hey @siong1987 , could you please take another look? Thanks!

**Comment by unordered-set** [comment]
Hey @siong1987  , could you please take another look? Thanks!

**Review by siong1987 — APPROVED** [review]

**Pull request merged**